domino_data/_feature_store/sync.py

```diff
@@ -168,17 +193,22 @@ def feature_store_sync(
     if not branch_name:
         branch_name = repo.active_branch.name
 
+    if not max_retries:
+        max_retries = 60
```

Comment:
wow that's a big number :D

Reply:
I simply give one minute for the users to try the locking. Do you have a good sense of how long waiting time is acceptable?

Reply:
oh is this 60 number the number of seconds? I thought it was number of retries

Reply:
It's the number of retries. After each retry, it sleeps for 1 second. So basically retry for 1 minute.